### PR TITLE
feature: add extraFilteredInterfaces configuration

### PR DESCRIPTION
**Pull request by juju812**
Allow users to custom interfaces that EdgeTunnel not listen to, and set interface "kube-ipvs" filtered by default.

### CODE REVIEW

**Comment by Poorunga** [comment]
Please fix DCO, you need add signoff to your commits

**Comment by juju812** [comment]
> Please fix DCO, you need add signoff to your commits

Done. :smile:

File: pkg/apis/config/v1alpha1/types.go [hunk 1]
```diff
@@ -277,9 +277,14 @@ type EdgeTunnelConfig struct {
...
 	// ListenInterfaces indicates the network interface devices that EdgeTunnel needs to listen to.
-	// empty or "*" stands for all netcards. You can also specify network interfaces such as "lo,eth0"
+	// empty or "*" stands for all NICs. You can also specify network interfaces such as "lo,eth0"
 	// default "*"
 	ListenInterfaces string `json:"listenInterfaces,omitempty"`
+	// ExtraFilteredInterfaces indicates user defined network interface devices that EdgeTunnel not to listen to.
+	// by default, interfaces prefixed with "docker", "edgemesh", "tunl", "flannel", "cni", "br" and "kube-ipvs" will be filtered.
+	// empty stands for no additional NICs to be filtered. You can specify network interfaces separated with comma such as "lo,eth0"
+	// default ""
+	ExtraFilteredInterfaces string `json:"extraFilteredInterfaces,omitempty"`
```
Comment (Poorunga) [inline pkg/apis/config/v1alpha1/types.go:287]: Are you willing to add the `ListenInterfaces and ExtraFilteredInterfaces` fields in the `build/helm/edgemesh/charts/agent/templates/04-configmap.yaml` file synchronously?

**Pull request merged**